elements/rh-table/rh-table.ts

```diff
@@ -138,6 +143,27 @@ export class RhTable extends LitElement {
     if (this.#table && this.#summary) {
       this.#table.setAttribute('aria-describedby', 'summary');
     }
+
+    /* Auto-assign `data-label` attributes based on column headers */
+    if (this.#table?.tHead && this.#colHeaders && this.#rows) {
```

Comment:
```suggestion
    if (this.#table?.tHead && this.#colHeaders.length && this.#rows) {
```

Reply:
@bennypowers `this.#colHeaders.length` I'm getting a TS error `Object is possibly 'undefined'` on this with the added `length`. Removing `length` for now, since it seems to work without it.

Reply:
You could use optional chaining and check with `this.#colHeaders?.length`

That would be the safest solution